parsec/core/local_device.py

```diff
@@ -54,6 +61,12 @@ class DeviceFileSchema(BaseSchema):
     human_handle = HumanHandleField(allow_none=True, missing=None)
     device_label = fields.String(allow_none=True, missing=None)
 
+    # Store device and organization ID in the device file
+    # For legacy versions, this information is available in the file name
+    device_id = DeviceIDField(allow_none=True, missing=None)
+    organization_id = OrganizationIDField(allow_none=True, missing=None)
+    root_verify_key_hash = fields.String(allow_none=True, missing=None)
```

Comment:
In this context `root_verify_key_hash` is the first 10 chars of the hex representation of the hash, so a string makes sense. We could store the actual hash of the rvk, I don't know if it's useful or not. 

Another way to approach the problem would be to store the `slug` instead since it already contains all this info. Any opinion about that?

Reply:
c809c86